app/backend/src/main/java/com/example/controllers/FamilyController.java

```diff
@@ -0,0 +1,64 @@
+package com.example.controllers;
+
+import com.example.models.Family;
+import com.example.services.FamilyService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("api/families")
+public class FamilyController {
+    @Autowired
+    private FamilyService familyService;
```

Comment:
AFAIK, Spring has become opiniated against using @Autowired annotations, opting in for a constructor dependency injection. Don't know which version of Spring we're using yet as I'm looking at this on the fly, but it would be good to use the latest, and conform to the latest standards if that's the case :D